documentation/spatialID.md

```diff
@@ -0,0 +1,83 @@
+# An Introduction to Spatial ID
+
+Spatial ID (or 空間ID/くうかんID/kūkan ID in Japanese) is a system for managing information in 3D space across time.[^1] Just as [pixels](https://en.wikipedia.org/wiki/Pixel) represent data in two-dimensional space, Spatial ID uses [voxels](https://en.wikipedia.org/wiki/Voxel), or virtual boxes, to map information in three-dimensional space.
+
+## ID Specifications
+In the Spatial ID system, each voxel has a unique ID number according to one of two identification specifications: SpatialID or Extended Spatial ID
+
+### 1. Spatial ID: 
+In the Spatial ID specification, Zoom Level `z` applies to both x, y, and f
+
+```
+{z}/{f}/{x}/{y}
+```
+where:
+- `z` is the Zoom Level
+- `x` is the Longitude Index
+- `y` is the Latitude Index
+- `f` is the Altitude Index
+
+### 2. Extended Spatial ID
+In the Extended Spatial ID specification the zoom level is split into two parameters: `hZoom` applies to x and y; `vZoom` applies to f
+
+```
+{hZoom}/{x}/{y}/{vZoom}/{f}
+```
+where:
+- `hZoom` is the Horizontal Zoom Level
+- `x` is the Longitude Index
+- `y` is the Latitude Index
+- `vZoom` is the Vertical Zoom Level
+- `f` is the Altitude Index
```

Comment:
This identifier is [`z`](https://github.com/trajectoryjp/spatial_id_go/blob/4a6932b2df628e47e6c4c302d80baa8ed9a54e8c/common/object/spatial_id.go#L18).
Please correct it.
The same applies to the others.